keycloakapi/src/main/kotlin/de/klg71/keycloakmigration/keycloakapi/model/RoleScopeMapping.kt

```diff
@@ -0,0 +1,12 @@
+package de.klg71.keycloakmigration.keycloakapi.model
+
+import java.util.UUID
+
+data class RoleScopeMapping(
```

Comment:
I guess we can't use the class `Role` here because RoleScopeMapping needs no attributes?

Reply:
But i think we should be able to use the class RoleListItem here, maybe we can rename it if you find something more suitable.

Reply:
done